django_unicorn/components/unicorn_view.py

```diff
@@ -186,6 +186,16 @@ class UnicornView(TemplateView):
     calls = []
 
     def __init__(self, **kwargs):
+        # set all these as instance variables instead of just class variables. Otherwise, calling UnicornView()
```

Comment:
I'm not sure if this was really required, but my tests were awkward without it. They'd have to call `construct_component(<args>)` instead of just `UnicornView(<args>)` but I assume in non-test it was working right.

Reply:
I'm pretty sure most of these class variables should be moved into the init, especially the ones that use `{}` because otherwise all instances will share the same dictionary which I'm pretty sure is not intended. Good catch.

Reply:
I did originally remove them from the class variables, but there is an odd check in Django views that says you can't set kwargs for a view that don't match a class variable: https://github.com/django/django/blob/main/django/views/generic/base.py#L89-L94 so I left the class variables but set the instance ones to override them.